server.js

```diff
@@ -2,10 +2,12 @@
 var express = require('express');
 var app = express();
 
+const PORT = process.env.PORT || 8080;
```

Comment:
what does this line do. I know what it does, just making sure you know what it does.

Reply:
like explain the process, env, port.

Reply:
ok its quite simple so:
 `const` is making a variable that cant be changed
 `PORT` is the name of the variable since its a constant i made everything capital so people know not to change it
`|| 8080` is if the first part is null it will make the PORT equal 8080

and finally
`process.env.PORT` I have no idea

Reply:
ok fr tho i think `process.env.PORT` is the port (?) that the hosting server uses and if we run the backend locally it will just be 8080

Reply:
So after linking our git repository on their hosting website, this line of code just sets up the server with them.